src/client/api/iconProvider.ts

```diff
@@ -60,7 +60,7 @@ export default class IconProvider {
   }
 
   private processLoadedTypes() {
-    Object.keys(ICON_DATA.data).forEach(t => {
+    Object.keys(ICON_DATA.data).forEach((t) => {
```

Comment:
My linter complains about single variables inside of parentheses, your prettier settings seem to go in the opposite direction. We might need to confer to make the environment consistent in different dev environments. Similarly, running `yarn fix` undoes all these changes for me.

Reply:
Strange, if I run `yarn fix` it actually reformats to wrap function parameters in parens. 
I think I noted in the PR description that prettier was out of date, and that some of the prettier-formatting diffs were probably due to this.

Reply:
Ideally all files affected by a linter or linter rule change would be modified and included in a changeset.

Reply:
Yes, we should probably get together after the meeting to make sure that `yarn fix` is idempotent in both of our environments. I imagine there is some configuration that can be included or modified in the repo that will fix the issue.

Reply:
After the latest commit and a fresh clone of this branch, `yarn fix` does not result in any changes.